streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -130,7 +134,7 @@ private String threadSensorPrefix() {
 
     public Map<String, String> threadLevelTagMap() {
         final Map<String, String> tagMap = new LinkedHashMap<>();
-        tagMap.put(THREAD_ID_TAG, threadName);
+        tagMap.put(THREAD_ID_TAG_0100_TO_23, threadName);
```

Comment:
should this have a check similar to https://github.com/apache/kafka/pull/7367/files#diff-8b364ed2d0abd8e8ae21f5d322db6564R269-R272 to determine if `THREAD_ID_TAG` or `THREAD_ID_TAG_0100_TO_23` should be the key?

Reply:
No, because this is on thread level. This will be done in another PR.